Makefile

```diff
@@ -61,7 +62,7 @@ image: imagebuilder
 	$(IMAGE_BUILDER) -t $(IMAGE_TAG) . $(IMAGE_BUILDER_OPTS)
 
 fmt:
-	@gofmt -l -w $(SRC)
```

Comment:
why is this not just `gofmt -l -w pkg` and `gofmt -l -w test` ?

Reply:
...and cmd?  Are these 2 separate lines?  Is that a viable syntax?

Reply:
no - the syntax would be 
`gofmt -l -w pkg && gofmt -l -w test`

Reply:
updated.